How to install?

Hey, looks really interesting, although I wasn't able to install it to try it out. Any clue? :)
Thanks!

Ok, thanks. I tried, but without success. 
First, I installed the qmake-qt4. After running make I had this output:
<pre>
rho@zenbook:~/Icon-Set-Forge-master/Code$ make
/usr/lib/x86_64-linux-gnu/qt4/bin/uic mainwindow.ui -o ui_mainwindow.h
make: /usr/lib/x86_64-linux-gnu/qt4/bin/uic: Program not found
make: *** [ui_mainwindow.h] Error 127
</pre>

Then I tried installing more qt4 tools by running a <code> sudo apt-get install qt4-designer </code> which solved the program not found error, but didn't run the app. I have this output:

<pre>
rho@zenbook:~/Icon-Set-Forge-master/Code$ make
/usr/lib/x86_64-linux-gnu/qt4/bin/uic mainwindow.ui -o ui_mainwindow.h
g++ -c -m64 -pipe -std=c++11 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I/usr/share/qt4/mkspecs/linux-g++-64 -I. -I/usr/include/qt4/QtCore -I/usr/include/qt4/QtGui -I/usr/include/qt4 -I. -I. -o iconsetbrowser.o view/iconsetbrowser.cpp
g++ -c -m64 -pipe -std=c++11 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I/usr/share/qt4/mkspecs/linux-g++-64 -I. -I/usr/include/qt4/QtCore -I/usr/include/qt4/QtGui -I/usr/include/qt4 -I. -I. -o main.o control/main.cpp
g++ -c -m64 -pipe -std=c++11 -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I/usr/share/qt4/mkspecs/linux-g++-64 -I. -I/usr/include/qt4/QtCore -I/usr/include/qt4/QtGui -I/usr/include/qt4 -I. -I. -o mainwindow.o view/mainwindow.cpp
</pre>

--- a/.gitignore
+++ b/.gitignore
@@ -33,3 +33,6 @@ Makefile*
 
 # Build directores
 build*
+
+# Output binaries
+Code/IconSetForge

--- a/AUTHORS
+++ b/AUTHORS
@@ -1,7 +1,7 @@
 Copyright 2014 Manuel Kehl (manuel-kehl.de) 
 	- core application: code, concept, documentation
 
-Copyright 2014 Delwink, LLC (delwink.com) - documentation
+Copyright 2014-2015 Delwink, LLC (delwink.com) - documentation
 
 Feel free to add your copyright notice and/or contribution description here, 
 if you have added something to Icon Set Forge.

--- a/INSTALL
+++ b/INSTALL
@@ -1,0 +1,22 @@
+Building the Application
+========================
+
+You will first need to satisfy the build dependencies of Icon Set Forge before
+compiling. You will need to install qmake for Qt 4 and Qt 4 Designer. On Debian
+GNU/Linux and derivative systems, this can be done with the following command:
+
+    # apt-get install qt4-qmake qt4-designer
+
+You can compile the program using qmake and GNU make:
+
+    $ cd Code/
+    $ qmake-qt4
+    $ make
+
+Installing the Application
+==========================
+
+After compiling the program, you can execute the following from the Code
+directory to install Icon Set Forge:
+
+    # make install

